.github/workflows/ci.yml

```
@@ -0,0 +1,288 @@
+      
+      - name: Setup Release Tools
+        if: steps.pr_check.outputs.should_release == 'true'
+        run: |
+          # Install goreleaser
+          curl -sL https://github.com/goreleaser/goreleaser/releases/latest/download/goreleaser_Linux_x86_64.tar.gz | tar xz
+          sudo mv goreleaser /usr/local/bin/
+      
+      - name: Determine Version
+        if: steps.pr_check.outputs.should_release == 'true'
+        id: version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:16: Double quote to prevent globbing and word splitting

(shellcheck)